routes/index.js

```diff
@@ -0,0 +1,91 @@
+require("dotenv").config();
+const express = require("express");
+const twilio = require("twilio")(process.env.ACCOUNTSID, process.env.AUTHTOKEN);
+const router = express.Router();
+const { signup, factorview, success } = require("../views/");
+const { User, Code } = require("../models");
+
+router.get("/", (_req, res, next) => {
+	try {
+		res.send(signup);
+	} catch (error) {
+		next(error);
+	}
+});
+
+router.post("/", async (req, res, next) => {
+	try {
+		const [user, _created] = await User.findOrCreate({
+			where: {
+				name: req.body.name,
+				email: req.body.email,
+				phone: req.body.phone,
+			},
+		});
+		const validTime = new Date();
+		validTime.setSeconds(validTime.getSeconds() + 30);
+		const code = await Code.create({
+			authCode: Math.floor(1000 + Math.random() * 9000),
+			valid_until: validTime,
+		});
+		await code.setUser(user);
+		const fromNum = process.env.FROMNUM;
+		const toNum = user.phone;
+		const messageBody = "your 2 factor authentication code is " + code.authCode;
+		await twilio.messages.create({
```

Comment:
I'd argue this actually shouldn't be awaited.   Immediately redirect them; the twilio message op can be async

Reply:
that makes a lot of sense